.github/workflows/release.yml

```diff
@@ -3,15 +3,13 @@ on: [workflow_dispatch] # Manual trigger
 jobs:
   build:
     runs-on: ubuntu-20.04
-    container:
-      image: eclipse-temurin:18-jdk
-      options: --user root
     steps:
-      - run: apt update && apt install git -y && git --version
-      - run: git config --global --add safe.directory /__w/fabric-language-kotlin/fabric-language-kotlin
```

Comment:
Be careful changing this, as it may break the changelog generation (needs a recent ish version of git). Updating to run on ubuntu-22.04 should mitatge any issues.

Reply:
Alright, I changed those as running with container seems to break the commit actions.